src/pages/resultsView/querySummary/QuerySummary.tsx

```diff
@@ -43,7 +43,7 @@ export default class QuerySummary extends React.Component<{ queryStore:QueryStor
 
     private get singleStudyUI() {
         return <div>
-            <h4><StudyLink study={this.props.store.studies.result[0]}/></h4>
+            <h4><StudyLink study={this.props.store.userQueryStudies.result[0]}/></h4>
```

Comment:
we need to be 100% sure we get all references to studies.  TS can really help us with that.   1. rename store.studies to something like nonVirtualStudies (@adamabeshouse comment?)  2. refactor studyIdToSummary to use that.   3. TS is going to complain for every remaining instance of studies.   You could then fix all the ts errors with the new userQueryStudies and be sure you're getting everything.    I would actually change userQueryStudies to just queriedStudies

Reply:
`userQueryStudies` is only used to render summary section on the query(mainly name and description). and `studies` contains physical studies in queried session. 
Anyways will rename `studies` to `physicalStudies` and `userQueryStudies` to `queriedStudies`

Reply:
nonVirtualStudies makes sense